crc/services/workflow_service.py

```diff
@@ -58,7 +58,7 @@ def make_test_workflow(spec_id):
 
     @staticmethod
     def delete_test_data():
-        for study in db.session.query(StudyModel).filter(StudyModel.user_uid=="test"):
```

Comment:
I've come to prefer the filter() syntax, as it forces you to be explicit,handles type checking, and works better with joins.